tools/docker/documentation/Dockerfile

```diff
@@ -1,3 +1,3 @@
-FROM python:2-stretch
+FROM python:3.6-stretch
```

Comment:
Is there a reason we chose 3.6 rather than e.g. 3.8? It seems like we'd be better taking the option that means we don't have to revisit this soon.

Reply:
Good point. By the way, let's try Sphinx 3 again (since https://bugs.python.org/issue40204 apparently landed some fixes in Py 3.8).